ui/raidboss/data/06-ew/trial/endsinger.ts

```diff
@@ -0,0 +1,231 @@
+      },
+      outputStrings: {
+        ne: Outputs.northeast,
+        nw: Outputs.northwest,
+        se: Outputs.southeast,
+        sw: Outputs.southwest,
+      },
+    },
+    {
+      id: 'Endsinger Elegeia',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: ['662C', '6682'], source: 'The Endsinger', capture: false }),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'Endsinger Fatalism',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6632', source: 'The Endsinger', capture: false }),
+      condition: (data) => data.storedStar?.x !== undefined,
+      delaySeconds: 15,
+      alertText: (data, _matches, output) => {
+        const x = data.storedStar?.x;
+        const y = data.storedStar?.y;
+
+        delete data.storedStar;
+        if (!x || !y)
+          return undefined;
+
+        if (x < 100) {
+          if (y < 100)
+            return output.se?.();
+
+          return output.ne?.();
+        }
+        if (y < 100)
+          return output.sw?.();
+
+        return output.nw?.();
+      },
+      outputStrings: {
+        ne: Outputs.northeast,
+        nw: Outputs.northwest,
+        se: Outputs.southeast,
+        sw: Outputs.southwest,
+      },
+    },
+    {
+      id: 'Endsinger Galaxias',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6C69', source: 'The Endsinger', capture: false }),
+      response: Responses.knockback(),
+    },
+    {
+      id: 'Endsinger Elenchos Middle',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6644', source: 'The Endsinger', capture: false }),
+      response: Responses.getOut(),
+    },
+    {
+      id: 'Endsinger Elenchos Outsides',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6642', source: 'The Endsinger', capture: false }),
+      response: Responses.getIn(),
+    },
+    {
+      id: 'Endsinger Death\'s Embrace',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6649', source: 'The Endsinger', capture: false }),
+      response: Responses.spread(),
+    },
+    {
+      id: 'Endsinger Death\'s Embrace Feathers',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6649', source: 'The Endsinger', capture: false }),
+      delaySeconds: 5.7,
+      response: Responses.moveAway(),
+    },
+    {
+      id: 'Endsinger Death\'s Aporrhoia',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '663D', source: 'The Endsinger', capture: false }),
+      infoText: (_data, _matches, output) => {
+        return output.awayFromHead?.();
+      },
+      outputStrings: {
+        awayFromHead: {
+          en: 'Away from head',
+        },
+      },
+    },
+    {
+      id: 'Endsinger Hubris',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6652', source: 'The Endsinger', capture: true }),
+      response: Responses.tankBuster(),
```

Comment:
My log has at least one instance of this being a `22` log line. Maybe it should be `tankCleave` instead?